### PR TITLE
feature/306_strings_translation

### PR DESCRIPTION
**Pull request by lucasrlt**
Add a translation file for strings.xml. 
The default language is now english and the app switches to French if it is the current locale.

Any comment translation changes/strings forgotten are appreciated 

I also added two minor fixes:
- The application's landscape mode is now disabled
- The weather block is now at the end of a tour view, not the beginning
- The app doesn't go back to the experience activity anymore when going back from a POI content fragment

**Ready to review**

### CODE REVIEW

**Review by ToufiPF — COMMENTED** [review]
Cool PR, I just feel like the string-fr file could be rearranged a bit (grouping the strings by activity/fragment a bit like in the original one), though if you don't have the time to do it it's ok.

**Comment by lucasrlt** [comment]
> Cool PR, I just feel like the string-fr file could be rearranged a bit (grouping the strings by activity/fragment a bit like in the original one), though if you don't have the time to do it it's ok.

Yeah you're right, I actually just used the Android studio translation manager which doesn't really sort things up. I'll structure the file in a new commit right now :smile:

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (aouame) [inline .idea/misc.xml:3]: you should restore the idea file. It should'nt be an issue later with Aurelien's pr

**Review by aouame — COMMENTED** [review]
Good pr. The fixes are nicely done and were necessary. Just a comment about the idea file.

**Review by aouame — DISMISSED** [review]

**Review by hadiyahya11 — DISMISSED** [review]
Nice pr. The translations are on point

**Review by ToufiPF — APPROVED** [review]

**Review by aouame — DISMISSED** [review]

**Review by ToufiPF — APPROVED** [review]

**Review by hadiyahya11 — APPROVED** [review]

**Pull request merged**